pyperformance/_manifest.py

```diff
@@ -170,14 +170,12 @@ def _add_benchmark(self, spec, metafile, resolve, filename):
     def _add_group(self, name, entries):
         if name in self._byname:
             raise ValueError(f'a group and a benchmark have the same name ({name})')
-        if name == 'all':
-            raise ValueError('a group named "all" is not allowed ("all" is reserved for selecting the full set of declared benchmarks)')
```

Comment:
This should still be the case.  So the Pyston benchmarks' manifest needs to be fixed.